### PR TITLE
Register APIService for apiregistration.k8s.io/v1beta1

### LINKED ISSUE #15516: etcdhelper tool cannot decode all types
Same underlying issue as #15511

I guess I need to make a test scheme that has all the types...

### PR DESCRIPTION
**Pull request by dmage**
```Fixes #15516```

### CODE REVIEW

File: tools/etcdhelper/etcdhelper.go [hunk 1]
```diff
@@ -19,9 +19,14 @@ import (
 	// install all APIs
 	_ "github.com/openshift/origin/pkg/api/install"
 	_ "github.com/openshift/origin/pkg/quota/apis/quota/install"
+	apiregistration "k8s.io/kube-aggregator/pkg/apis/apiregistration/install"
 	_ "k8s.io/kubernetes/pkg/api/install"
 )
 
+func init() {
+	apiregistration.Install(api.GroupFactoryRegistry, api.Registry, api.Scheme)
```
[the remark this answers is not in the document]
Comment (deads2k) [reply tools/etcdhelper/etcdhelper.go]: > @deads2k @sttts is there a better way to handle this?

You should create your own scheme, but no, not really.   You need protobuf decoding information from all groups.  If the API server had its own scheme (help wanted), then you'd be able to load that one.

[the remark this answers is not in the document]
Comment (mfojtik) [reply tools/etcdhelper/etcdhelper.go]: So this is probably OK for now to get us out from seeing garbage in tests/logs. Once API server gets its scheme, we can nuke this.

**Comment by enj** [comment]
@dmage this is fine for now but I want the issue to stay open so I commented out your `Fixes`.  In general for tests I wanted to make a custom scheme and register the whole world into it.  Its not perfect and still manual work to keep it up to date, but at least the etcd storage test would not miss types.

File: tools/etcdhelper/etcdhelper.go [hunk 2]
```diff
@@ -151,7 +156,7 @@ func dump(client *clientv3.Client) error {
 	for _, kv := range response.Kvs {
 		obj, _, err := decoder.Decode(kv.Value, nil, nil)
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "WARN: error decoding value %s: %v\n", string(kv.Value), err)
+			fmt.Fprintf(os.Stderr, "WARN: error decoding value %q: %v\n", string(kv.Value), err)
```
Comment (sttts) [inline tools/etcdhelper/etcdhelper.go]: the key would be helpful in the output.

**Pull request merged**